tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>lacking a comma here: the output signal, and the output unit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>